src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1424,9 +1424,9 @@ def create_vm(vm_name, resource_group_name, image=None,
               public_ip_address=None, public_ip_address_allocation='dynamic',
               public_ip_address_dns_name=None,
               os_disk_name=None, os_type=None, storage_account=None,
-              storage_caching='ReadWrite', storage_container_name='vhds',
+              storage_caching=None, storage_container_name=None,
```

Comment:
Any reason we like to change this? I thought it is recommended by putting default value in a function's signature rather in param.py.